webapp/components/post_view/post_info/post_info.jsx

```diff
@@ -239,10 +240,16 @@ export default class PostInfo extends React.PureComponent {
             );
         }
 
+        // timestamp should not be a permalink if the post has been deleted, is ephemeral message, or is pending
+        const isPermalink = !(PostUtils.isSystemMessage(this.props.post)
```

Comment:
I think it's fine for this permalinks to show for regular system messages, just not ephemeral ones. You can check that with https://github.com/mattermost/mattermost-redux/blob/master/src/utils/post_utils.js#L60

Reply:
Ah. I didn't realize there was a difference